src/ros_filter_bias_estimator.cpp

```diff
@@ -263,13 +274,29 @@ void RosFilterBiasEstimator::updateBiasEstimate(Eigen::Vector3d &orientation_mea
                 // Cannot create the pose measurement yet - there is no offset information or it has been flagged as invalid
                 is_valid[axis] = false;
             }
+
+            // Calculate the difference between estimates and place in the vector
+            double filter_difference = uncorrected_orientation_estimate_[axis] - orientation_estimate[axis];
+            if(estimate_differences_.size() <= current_variance_estimation_slot_)
+            {
+                Eigen::Vector3d new_diff(0, 0, 0);
+                estimate_differences_.push_back(new_diff);
+            }
+            estimate_differences_[current_variance_estimation_slot_][axis] = filter_difference;
```

Comment:
Can you forsee a reason where there would be a benefit for `filter_difference`/displacement to be angle wrapped or normalized some other way?

Reply:
Yeah, it could be angle wrapped. I'll add that to be safe.

Reply:
Angle wrapped